validator/testcases/javascript/predefinedentities.py

```diff
@@ -36,6 +45,11 @@
     "description":
         "Changing network preferences may be dangerous, and often leads to "
         "performance costs.",
+    "signing_help":
+        "Changes to these preferences are strongly discouraged. If at all "
+        "possible, you should remove any reference to them from "
+        "your extension. Extensions which do modify these preferences "
+        "must undergo one round of light manual review.",
```

Comment:
"At least the first submission of any add-on which modifies these preferences must undergo manual review."